### PR TITLE
W-15424324-IDP-ConnectedApp-Credentials-se

### PR DESCRIPTION
**Pull request by sebermuley**
Added Connected App Credentials for MuleSoft IDP via RPA Builder

### CODE REVIEW

**Comment by sebermuley** [comment]
Please SME review. Thank you.

**Review by Keburichi — DISMISSED** [review]

**Comment by sebermuley** [comment]
Please peer review. Thank you.

File: rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-credential.adoc [hunk 1]
```diff
@@ -88,14 +91,24 @@ The credential can be deployed in the test phase.
...
+** *Connected App Credentials*
++
+Use this credential to access MuleSoft Intelligent Document Processing (IDP) via RPA Builder action steps.
++
+*** *Client ID*
++
+Client ID of the connected app configured to call IDP.
+*** *Client Secret*
++
+Client Secret of the connected app configured to call IDP
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-credential.adoc]:
```suggestion
Client Secret of the connected app configured to call IDP.
```

File: rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-credential.adoc [hunk 2]
```diff
@@ -88,14 +91,24 @@ The credential can be deployed in the test phase.
...
 +
 Use this credential to access Amazon Web Services. Fill in the client ID, client secret, and region. The session token is optional.
+** *Connected App Credentials*
++
+Use this credential to access MuleSoft Intelligent Document Processing (IDP) via RPA Builder action steps.
++
+*** *Client ID*
++
+Client ID of the connected app configured to call IDP.
+*** *Client Secret*
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-credential.adoc:109]:
```suggestion

*** *Client Secret*
```

**Review by Crispy-Salesforce — DISMISSED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**